.github/workflows/ci.yml

```
@@ -21,14 +21,44 @@ jobs:
+
+  test:
+    needs: run
+    runs-on: ubuntu-24.04
+
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/download-artifact@v4
+      with:
+        name: sim_output
+        path: tests/run/
+        if-no-files-found: error
```

actionlint (1.7.4)

input "if-no-files-found" is not defined in action "actions/download-artifact@v4". available inputs are "github-token", "merge-multiple", "name", "path", "pattern", "repository", "run-id"

(action)